src/Words.js

```diff
@@ -1,5377 +1,5793 @@
-  ["priest", "n", 14889],
-  ["amazing", "j", 15124],
-  ["intense", "j", 14452],
-  ["advance", "n", 14645],
-  ["employer", "n", 14846],
-  ["shock", "n", 14551],
-  ["inspire", "v", 14771],
-  ["adjust", "v", 14564],
-  ["retire", "v", 14979],
-  ["sure", "r", 14854],
-  ["visible", "j", 14691],
-  ["kiss", "v", 16128],
-  ["illness", "n", 14726],
-  ["cap", "n", 14672],
-  ["habit", "n", 14501],
-  ["competitive", "j", 14938],
-  ["juice", "n", 15388],
-  ["congressional", "j", 14936],
-  ["involvement", "n", 15640],
-  ["dominate", "v", 14768],
-  ["previously", "r", 15092],
-  ["whenever", "r", 14484],
-  ["transfer", "v", 14686],
-  ["analyze", "v", 15587],
-  ["another", "p", 14441],
-  ["attach", "v", 14552],
-  ["for", "c", 14594],
-  ["indian", "n", 15021],
-  ["disaster", "n", 14684],
-  ["parking", "n", 14970],
-  ["prospect", "n", 14521],
-  ["boss", "n", 14587],
-  ["complaint", "n", 14551],
-  ["championship", "n", 16008],
-  ["coach", "v", 15910],
-  ["exercise", "v", 14766],
-  ["fundamental", "j", 15199],
-  ["severe", "j", 14609],
-  ["enhance", "v", 15393],
-  ["mystery", "n", 14211],
-  ["impose", "v", 14881],
-  ["poverty", "n", 14891],
-  ["other", "i", 14165],
-  ["entry", "n", 14643],
-  ["fat", "j", 14826],
-  ["spending", "n", 14965],
-  ["king", "n", 14445],
-  ["evaluate", "v", 15780],
-  ["symbol", "n", 14596],
-  ["still", "j", 14308],
-  ["trade", "v", 14624],
-  ["maker", "n", 14586],
-  ["mood", "n", 14227],
-  ["accomplish", "v", 14325],
-  ["emphasis", "n", 15337],
-  ["illustrate", "v", 15091],
-  ["boot", "n", 15033],
-  ["monitor", "v", 14499],
-  ["asian", "j", 14873],
-  ["entertainment", "n", 14615],
-  ["bean", "n", 14996],
-  ["evaluation", "n", 16213],
-  ["creature", "n", 14852],
-  ["commander", "n", 14425],
-  ["digital", "j", 15048],
-  ["arrangement", "n", 14403],
-  ["concentrate", "v", 14155],
-  ["total", "n", 14559],
-  ["usual", "j", 14305],
-  ["anger", "n", 14241],
-  ["psychological", "j", 15249],
-  ["heavily", "r", 13964],
-  ["peak", "n", 14481],
-  ["approximately", "r", 15172],
-  ["increasing", "j", 14886],
-  ["disorder", "n", 14780],
-  ["missile", "n", 15147],
-  ["equally", "r", 14389],
-  ["vary", "v", 14954],
-  ["wire", "n", 14310],
-  ["round", "n", 14539],
-  ["distribution", "n", 15249],
-  ["transportation", "n", 14516],
-  ["holy", "j", 14090],
-  ["ring", "v", 14757],
-  ["twin", "n", 14610],
-  ["command", "n", 14022],
-  ["commission", "n", 14371],
-  ["interpretation", "n", 15613],
-  ["breakfast", "n", 14397],
-  ["stop", "n", 14117],
-  ["strongly", "r", 14459],
-  ["engineering", "n", 14853],
-  ["luck", "n", 14186],
-  ["socalled", "j", 14065],
-  ["constant", "j", 14069],
-  ["race", "v", 14233],
-  ["clinic", "n", 14016],
-  ["veteran", "n", 14366],
-  ["smell", "v", 15214],
-  ["tablespoon", "n", 15804],
-  ["capable", "j", 13933],
-  ["nervous", "j", 14100],
-  ["tourist", "n", 13895],
-  ["light", "v", 14720],
-  ["toss", "v", 14565],

```

Comment:
```suggestion
    this.wordFrequency = new Map(); // TODO: this is used in sorted value order, consider using http://www.collectionsjs.com/sorted-map
```

Reply:
Just meant as a documentation note for now, no need to actually do anything other than the suggested comment change.